src/Tizen.NUI/src/public/PixelData.cs

```diff
@@ -76,6 +76,21 @@ public enum ReleaseFunction
             DeleteArray
         }
 
+        /// <summary>
+        /// Get URL from pixel data.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Url
```

Comment:
StyleCop(.NET Framework Design Guideline) recommends to use [Uri](https://docs.microsoft.com/en-us/dotnet/api/system.uri?view=net-5.0) Type instead of string~
Please check [CA1056](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1056) StyleCop warnings(it is the same as the warning below.)

Reply:
Thank you I fixed it